logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -137,6 +137,11 @@ interface UserConfigRepository {
     fun setShouldFetchE2EITrustAnchors(shouldFetch: Boolean)
     fun getShouldFetchE2EITrustAnchor(): Boolean
     suspend fun setTrackingIdentifier(identifier: String)
+    suspend fun getTrackingIdentifier(): String?
+    suspend fun observeTrackingIdentifier(): Flow<Either<StorageFailure, String>>
```

Comment:
```suggestion
    suspend fun setCurrentTrackingIdentifier(newIdentifier: String)
    suspend fun getCurrentTrackingIdentifier(): String?
    suspend fun observeCurrentTrackingIdentifier(): Flow<Either<StorageFailure, String>>
```

Question: I like it and I understand it, but, What if... we do something like this ☝️  would it make it clearer 🤔 
(adding the Current word to it) ?

Reply:
Oh for sure!